### PR TITLE
Moves config.py and config.yaml into bigbang dir. Addresses #569

### PR DESCRIPTION
**Pull request by sbenthall**
This PR addrsses issue #569 by moving the `config/` directory contents into the `bigbang/` directory, as per @MridulS 's recommendation.

The one thing I couldn't figure out was how fix some of the pipermail ingress tests. These depended on certificates that for some reason were bundled in the `config/` directory. @Christovis , please advise.

### CODE REVIEW

**Comment by sbenthall** [comment]
This PR now passes tests and I believe will fix issue #569 .  

I know it doesn't do all the refactoring recommended by @MridulS in that issue...

@MridulS any chance you could review this PR ?

cc @ccosborne this will soon be merged in.

**Comment by ccosborne** [comment]
Thanks @sbenthall !

File: tests/analysis/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    if match is not None:
+        return re.search(domain_regex, match[0])[1]
+
+    else:
+        return None
+
+
+"""
+class TestExtractEmail(unittest.TestCase):
```
Comment (Christovis) [inline tests/analysis/test_utils.py:40]: Are we going to need this commented-out code later/elsewhere or could we remove it?

**Review by Christovis — COMMENTED** [review]
As all tests are passed and the changed made make sense to me too, I would say we can merge it after my question regarding the commented-out code is adressed :)

File: tests/analysis/test_utils.py (hunk 1 above)
Comment (sbenthall) [reply tests/analysis/test_utils.py:40]: I see -- I'll remove it and then merge this PR. Thanks!

Comment (sbenthall) [reply tests/analysis/test_utils.py:40]: Hmmm. Actually -- this test file as written does not contain any tests.
The uncommented code here is duplicate code of functions in `bigbang.analysis.utils`.
It looks like somebody was going to write tests for those functions, but did not complete the job.

Comment (sbenthall) [reply tests/analysis/test_utils.py:40]: oh -- this whole file is some work in progress I should not have checked in. my bad

**Pull request merged**